packages/grid/x-data-grid/src/models/api/gridEditingApi.ts

```diff
@@ -192,9 +192,16 @@ interface GridStopRowEditModeParams {
    */
   ignoreModifications?: boolean;
   /**
-   * The field from the row below to move the focus after updating the row.
+   * The field that has focus when the editing is stopped.
+   * Used to calculate which cell to move the focus after finishing editing.
    */
-  fieldFromRowBelowToFocus?: string;
+  field?: string;
+  /**
+   * To which cell to move focus after finishing editing.
+   * Only works if the field is also specified.
```

Comment:
Maybe describe what is the behavior for the focus when the field is not specified ?